server/chat-plugins/poll.ts

```diff
@@ -377,6 +434,14 @@ export const commands: ChatCommands = {
 				poll.timeout = setTimeout(() => {
 					if (poll) poll.end();
 					room.minorActivity = null;
+					if (!room.queuedActivity) room.queuedActivity = [];
+					if (room.queuedActivity.length) {
+						room.minorActivity = room.queuedActivity[0];
+						this.addModAction(`The queued poll was started.`);
+						this.modlog(`POLL`, null, `queued`);
+						room.minorActivity.display();
+						room.queuedActivity.splice(0, 1);
+					}
```

Comment:
```suggestion
					if (room.queuedActivity?.length) {
						room.minorActivity = room.shift()!;
						this.addModAction(`The queued poll was started.`);
						this.modlog(`POLL`, null, `queued`);
						room.minorActivity.display();
						if (!room.queuedActivity.length) room.queuedActivity = null;
					}
```

Reply:
Actually, this should be put in its own function, `room.checkActivityQueue`. You use it in multiple places.